test/sqllogictest/chbench.slt

```diff
@@ -508,28 +507,26 @@ ORDER BY revenue DESC
 Explained Query:
   Finish order_by=[#1 desc nulls_first] output=[#0, #1]
     Reduce group_by=[#1] aggregates=[sum(#0)] // { arity: 2 }
-      Project (#12, #19) // { arity: 2 }
-        Join on=(#0 = #7 AND #1 = #5 = #9 AND #2 = #6 = #10 = #14 AND #3 = #17 = #18 AND #4 = #8 AND #11 = #13 AND #15 = #16 AND #20 = #21) type=delta // { arity: 22 }
+      Project (#16, #24) // { arity: 2 }
+        Join on=(#0 = #7 AND #1 = #5 = #9 AND #2 = #6 = #10 = #19 AND #3 = #22 = #23 AND #4 = #8 AND #12 = #18 AND #20 = #21 AND #25 = #26) type=delta // { arity: 27 }
           implementation
             %0:customer » %5:nation[#0]UK » %6:region[#0]UKef » %1:order[#2, #1, #3]KKKAif » %2:orderline[#2, #1, #0]KKKA » %3:stock[#0, #1]UKK » %4:supplier[#0, #1]UKK
             %1:order » %0:customer[#0..=#2]UKKK » %5:nation[#0]UK » %6:region[#0]UKef » %2:orderline[#2, #1, #0]KKKA » %3:stock[#0, #1]UKK » %4:supplier[#0, #1]UKK
             %2:orderline » %1:order[#0..=#2]UKKKif » %0:customer[#0..=#2]UKKK » %3:stock[#0, #1]UKK » %4:supplier[#0, #1]UKK » %5:nation[#0]UK » %6:region[#0]UKef
-            %3:stock » %4:supplier[#0]UK » %5:nation[#0]UK » %6:region[#0]UKef » %0:customer[#2, #3]KK » %1:order[#2, #1, #3]KKKAif » %2:orderline[#0..=#3]KKKK
+            %3:stock » %4:supplier[#0]UK » %5:nation[#0]UK » %6:region[#0]UKef » %0:customer[#3]KA » %1:order[#2, #1, #3]KKKAif » %2:orderline[#2, #1, #0]KKKA
             %4:supplier » %5:nation[#0]UK » %6:region[#0]UKef » %0:customer[#3]KA » %1:order[#2, #1, #3]KKKAif » %2:orderline[#2, #1, #0]KKKA » %3:stock[#0..=#2]UKKK
             %5:nation » %6:region[#0]UKef » %0:customer[#3]KA » %1:order[#2, #1, #3]KKKAif » %2:orderline[#2, #1, #0]KKKA » %3:stock[#0, #1]UKK » %4:supplier[#0, #1]UKK
             %6:region » %5:nation[#2]KA » %0:customer[#3]KA » %1:order[#2, #1, #3]KKKAif » %2:orderline[#2, #1, #0]KKKA » %3:stock[#0, #1]UKK » %4:supplier[#0, #1]UKK
-          ArrangeBy keys=[[#0..=#2], [#2, #3], [#3]] // { arity: 4 }
+          ArrangeBy keys=[[#0..=#2], [#3]] // { arity: 4 }
```

Comment:
1 less arrangement. (Arrangements being built in the new plan are a subset of arrangements in the old plan.)